src/firefly/js/visualize/ui/WorldPtFormat.jsx

```diff
@@ -29,7 +29,7 @@ export function formatWorldPt(wp, pad=3, useBold=true) {
     }
 }
 
-export function formatWorldPtToString(wp,addNewLIne=false) {
+export function formatWorldPtToString(wp,dcmOrHms,useResolverName=true,addNewLIne=false) {
```

Comment:
`formatWorldPtToString()` should reverted so you just use `formatWorldPtToStringSimple`. The way  if wp.objName is defined that is returned alone.
Make `formatWorldPtToStringSimple` take the `dcmOrHms` parameter. I think if should be a boolean such as `dcmOnly`. 

I would like you to change both functions to also take wp as a string.

first two lines in both, something like:
```js
const actualWp= isValidPoint(wp) ? wp : isString(wp) : parseWorldPt(wp) : undefined;
if (!isValidPoint(actualWp)) return '';
```

Reply:
Thanks - this simplifies it!

> I think if should be a boolean such as dcmOnly.

I thought about doing it but boolean is not a good type to represent 3 states we have: DCM only, HMS only, DCM and HMS both (it's ambiguous if false means both or HMS only). I instead created an enum type (`EQ_TYPE`) and I'm using it everywhere now instead of hard coded strings.